resources/public/api/conf/2.0/examples/retrieveBalanceAndTransactions/itemDetails.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
would it be easier to have items in it as well?

Reply:
Will probably need to merge all the examples into one file, so will update after next commit